src/Rings/mpoly-ideals.jl

```diff
@@ -1126,6 +1126,12 @@ julia> L = minimal_primes(I)
 function minimal_primes(I::MPolyIdeal; algorithm::Symbol = :GTZ, cache::Bool=true)
   has_attribute(I, :minimal_primes) && return get_attribute(I, :minimal_primes)::Vector{typeof(I)}
   R = base_ring(I)
+  if coefficient_ring(R) isa QQField && is_zero(dim(I))
```

Comment:
`dim(I)` triggers a GB computation?

Reply:
Triage suggest this would perhaps make sense with `is_known_to_be_zero_dimensional(I)` check -- i.e. if we know this information, we can call the 0-dim method.

But otherwise, leave it singular -- if it detects the 0-dim case it still handles it well.